datafusion/physical-expr-common/src/binary_view_map.rs

```diff
@@ -178,8 +183,12 @@ where
     /// `make_payload_fn`:  invoked for each value that is not already present
     /// to create the payload, in order of the values in `values`
     ///
-    /// `observe_payload_fn`: invoked once, for each value in `values`, that was
-    /// already present in the map, with corresponding payload value.
+    /// `update_payload_fn`: invoked when the value is already present in the map.
```

Comment:
Re-reading this code i wonder why the term "payload" is used rather than "value" -- it is a Map after all, so it has keys and values 🤔 

Maybe it is because the `values` was already used.

Reply:
Yep, that might be the case. It could make more sense to stick with the more traditional key/value naming.